src/pages/guides/eqp/v1/access-keys.md

```diff
@@ -0,0 +1,65 @@
+-  API access keys are specific to each **Marketplace Developer Portal** environment.  API access keys generated for the **sandbox** cannot be used for **production**, and vice-versa.
+-  For each **Marketplace Developer Portal** environment, you have a limit of three (3) API access keys.
+
+## How do I create an API access key
+
+-  You create your API access key from one, or both, of the **Marketplace Developer Portal** user interfaces:
+   -  production - [https://developer.magento.com][1]
+   -  sandbox - [https://developer-stg.magento.com][2]
+
+### Create a new API access key
+
+1. From the **Marketplace Developer Portal**, sign in, click on your name (top, right corner), and choose either the **Account Information** or the **Marketplace Profile** link.
+
+1. From the left-hand side navigation menu, click on **Manage API Keys**.
+
+1. If you are **not** [eligible](../v1/index.md#api-eligibility) you will see the following screen.  You will **not** be able to create an API access key, and thus you will **not** be able to use the **Marketplace EQP API**.
+   ![Not Eligible to Use the Marketplace EQP API](../_images/sandbox-not-eligible.png)
+
+1. If you **are** eligible to use the **Marketplace EQP API**, you will see the following screen.
+   ![Empty List of API Access Keys](../_images/sandbox-access-key-empty.png)
+
+1. Click **Create API Access Key**.
+
+1. In the "Create New API Key" dialog, enter an **API Key Name**.  This name is for your own use. Then, click **Continue**.
+   ![Create API Access Key Dialog](../_images/sandbox-access-key-create.png)
+
+1. Your new API access key appears in the list.  Notice that your API access key has an associated **application ID** and an **application secret**.
+   ![New API Access Key Added](../_images/sandbox-access-key-list.png)
+
+## Manage your API access keys
+
+-  From the list of your API access keys, you can **regenerate** the key.
+   -  Regenerating the key will result in a new pair of the **application ID** and **application secret**.
+   -  Once regenerated, the previous pair can no longer be used to obtain a [session token](auth.md#how-to-use-a-session-token).
+   -  However, any **session tokens** obtained from the previous key will remain valid, until those session tokens expire.
+-  From the list of your API access keys, you can **delete** the key.
+   -  Once deleted, the key can no longer be used to obtain a [session token](auth.md#how-to-use-a-session-token).
+   -  However, any **session tokens** obtained from the previous key will remain valid, until those session tokens expire.
+
+<InlineAlert variant="info" slots="text"/>
+
+Marketplace EQP API **access keys** generated in the [sandbox][2] environment are separate from those generated in [production][1].
+You cannot use access keys from one environment in the other.
+<br/><br/>
+**Session tokens** generated in the **sandbox** are separate from those generated in **production**.
```

Comment:
This isn't rendering the way I think you want it to.

![Screen Shot 2022-04-01 at 2 10 48 PM](https://user-images.githubusercontent.com/13662379/161326822-5f0cfb53-e90f-4311-93e3-9ce2d0780672.png)